lib/facade_converter.ts

```diff
@@ -146,6 +181,33 @@ export class FacadeConverter extends base.TranspilerBase {
     this.emit(')');
   }
 
+  private stdlibTypeReplacements: ts.Map<string> = {
+    'Date': 'DateTime',
+    'Array': 'List',
+
+    // Dart has two different incompatible DOM APIs
+    // https://github.com/angular/angular/issues/2770
+    'Node': 'dynamic',
+    'Text': 'dynamic',
+    'Element': 'dynamic',
+    'HTMLElement': 'dynamic',
+    'HTMLStyleElement': 'dynamic',
+    'HTMLInputElement': 'dynamic',
+    'HTMLDocument': 'dynamic',
+    'History': 'dynamic',
+    'Location': 'dynamic',
```

Comment:
Can you tack on 'XMLHttpRequest': 'HttpRequest' here so I can close PR #231 that would conflict with this change (as well as issue #230)?